### PR TITLE
chore: Delete tsbuildinfo after pack

### PR DESCRIPTION
**Pull request by kriskowal**
Cleans up more dross after building packages for npm.

```sh
#!/bin/bash
set -ueo pipefail
while read -r PKGJSON; do
  echo "$PKGJSON"
  jq '.scripts.postpack = $postpack' \
    --arg postpack "git clean -f '*.d.ts*' '*.tsbuildinfo'" \
    "$PKGJSON" |
    sponge "$PKGJSON"
done < <(
  npm query '.workspace:not([private])' |
  jq -r '.[].location | "\(.)/package.json"'
)
```

### CODE REVIEW

File: packages/base64/package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   "scripts": {
     "build": "exit 0",
     "prepack": "tsc --build tsconfig.build.json",
-    "postpack": "git clean -f '*.d.ts*'",
+    "postpack": "git clean -f '*.d.ts*' tsconfig.build.tsbuildinfo",
```
Comment (turadg) [inline packages/base64/package.json:36]: might as well delete any future ones
```suggestion
    "postpack": "git clean -f '*.d.ts*' *.tsbuildinfo",
```

Comment (kriskowal) [reply packages/base64/package.json:36]: Any thoughts on whether to defer the glob expansion to git?

Comment (turadg) [reply packages/base64/package.json:36]: nope

Comment (turadg) [reply packages/base64/package.json:36]: I've developed an opinion: https://github.com/Agoric/agoric-sdk/pull/9795/commits/e119569b394e27cdee6718dfafddc04ba90cfaed

hazarded a guess that git might do it a little faster

Comment (kriskowal) [reply packages/base64/package.json]: Following your lead. The other effect is that it will match files in subdirectories.

**Review by turadg — APPROVED** [review]

**Pull request merged**